docker-admin-ui/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14.19.3-alpine3.16 AS builder
+FROM node:14.20-alpine3.16 AS builder
```

Comment:
We have policy to use explicit version for base image. Please change `node:14.20-alpine3.16` to `node:14.20.0-alpine3.16`.